AISC-understanding-search/maze-transformer
# #88 - random paths

Resolves #88 
 
Changes:
- `create_dataset` and `test_generation` scripts will now generate maze tokenizations with random paths 
- Tiny update to notebook setup instructions

To test:
- Create a new dataset
- Run `eval_model` notebook with the new dataset. You should see that the paths are now random in the plotted mazes

<img width="438" alt="Screenshot 2023-03-15 at 11 47 26 AM" src="https://user-images.githubusercontent.com/7400942/225364746-24234f6e-9c34-4ce7-86ae-69d2418672a1.png">
